src/tools/perf/libperf.c

```diff
@@ -428,6 +464,27 @@ static ucs_status_t uct_perf_test_check_capabilities(ucx_perf_params_t *params,
         }
     }
 
+    if ((UCT_PERF_DATA_LAYOUT_ZCOPY == params->uct.data_layout)) {
+        if (params->iovcnt > max_iov) {
+            if ((params->flags & UCX_PERF_TEST_FLAG_VERBOSE) || !params->iovcnt) {
+                ucs_error("Wrong number of IOV entries. Requested is %lu,"
+                          " should be in the range 1...%lu", params->iovcnt, max_iov);
+            }
+            return UCS_ERR_UNSUPPORTED;
+        }
+        /* if iovcnt == 1 the message size checked above */
+        if ((UCX_PERF_CMD_AM == params->command) && (params->iovcnt > 1)) {
+            if (params->am_hdr_size > params->msg_size_lst[0]) {
```

Comment:
Of course we can. It is required changes in overall test logic related to this header. If you think it is important, let's make these changes in next PR.